Components/Grids/grids.html

```diff
@@ -0,0 +1,22 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="grids.css" />
+    <title>Grids</title>
+  </head>
+  <body>
+    <div class="grid-two">
+      <div>First column</div>
+      <div>Second column</div>
+    </div>
+
+    <div class="grid-three">
+      <div>First column</div>
+      <div>Second column</div>
+      <div>Second column</div>
```

Comment:
*Third Column

Reply:
Hey, yes, I've fixed that. Thanks for pointing it out Tejas.